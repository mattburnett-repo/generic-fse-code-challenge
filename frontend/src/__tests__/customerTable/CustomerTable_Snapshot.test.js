

// https://github.com/testing-library/jest-dom

import {renderer, MockedProvider, Customers, mockCustomerOperation} from '../../test/genericFseTestConfig'
import 'jest-canvas-mock';

describe('CustomerTable snapshot', () => {
    it('creates a snapshot', () => {
        const tree = renderer.create(
            <MockedProvider mocks={[mockCustomerOperation]} addTypename={false}>
                <Customers />
            </MockedProvider>
        )
        expect(tree).toMatchSnapshot();  
    });
}) // end PolicyTable snapshot