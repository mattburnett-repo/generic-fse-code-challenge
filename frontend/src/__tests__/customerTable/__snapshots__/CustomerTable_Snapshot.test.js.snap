// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomerTable snapshot creates a snapshot 1`] = `
Array [
  <div
    className="spinner"
  />,
  <div
    aria-label="loading-text"
    className="spinnerText"
    role="presentation"
  >
    ... loading
  </div>,
]
`;
